src/Microsoft.ML.Data/Depricated/Instances/HeaderSchema.cs

```diff
@@ -151,7 +151,7 @@ private static VersionInfo GetVersionInfo()
                 loaderAssemblyName: typeof(FeatureNameCollection).Assembly.FullName);
         }
 
-        public static void Save(ModelSaveContext ctx, in VBuffer<ReadOnlyMemory<char>> names)
+        internal static void Save(ModelSaveContext ctx, in VBuffer<ReadOnlyMemory<char>> names)
```

Comment:
>internal [](start = 8, length = 8)

private protected?

Reply:
The class is internal, so I think I should actually revert this change.

---
In reply to: [254963619](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254963619) [](ancestors = 254963619)